data/data/ovirt/bootstrap/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "ignition_bootstrap" {
+  description = "The ID of storage domain"
+  default     = ""
```

Comment:
please reconsider is empty is actually a valid value, similarly others

Reply:
done